wp-content/themes/csisnuclearnetwork/assets/_scss/components/_footnotes.scss

```diff
@@ -26,6 +23,13 @@
   }
 }
 
+.qtip-content {
+  a {
+    color: $color-accent-primary-blue-200 !important;
+    text-decoration: underline !important;
+  }
+}
+
 .footnotes {
   clear: both;
   margin: rem(40px) 0;
```

Comment:
Check the top and bottom margins in Figma.  It looks like the top should be 56 and the bottom should be 48 (combining the two margins).